### PR TITLE
LPS-179232 | Adjust Autom. test ClayAlertToast#MultipleContainersCanBeDisplayed

### PR DESCRIPTION
**Pull request by deborasoliveira**
**ClayAlertToast#MultipleContainersCanBeDisplayed**
**Test Plan**

- Given Clay Sample Portlet
- When Click info button
- And Click error button
- And Click success button
- Then Each toast appears from bottom to top

**JIRA Ticket** 
https://issues.liferay.com/browse/LPS-179232

### CODE REVIEW

**Comment by deborasoliveira** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:frontend

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase [hunk 1]
```diff
@@ -244,26 +244,72 @@ definition {
...
+
+			var positionAlertSuccess = ClayAlertToast.getElementRectanglePositionY(
+				code = "#ToastAlertContainer .alert-success",
+				index = 0);
+			var positionAlertFail = ClayAlertToast.getElementRectanglePositionY(
+				code = "#ToastAlertContainer .alert-danger",
+				index = 0);
+			var positionAlertInfo = ClayAlertToast.getElementRectanglePositionY(
+				code = "#ToastAlertContainer .alert-info",
+				index = 0);
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase:262]: Hey @deborasoliveira can you create a macro with this and use  3 times?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase [hunk 2]
```diff
@@ -244,26 +244,72 @@ definition {
...
+			var positionAlertInfo2 = ClayAlertToast.getElementRectanglePositionY(
+				code = "#ToastAlertContainer .alert-info",
+				index = 0);
+
+			if (${positionAlertSuccess} != ${positionAlertSuccess2}) {
+				takeScreenshot();
+
+				fail("FAIL. Success toast alert coordinate positions do not match.");
+			}
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase:298]: same here

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase [hunk 3]
```diff
@@ -244,26 +244,72 @@ definition {
...
+				fail("FAIL. Fail toast alert coordinate positions do not match.");
+			}
+
+			if (${positionAlertInfo} != ${positionAlertInfo2}) {
+				takeScreenshot();
+
+				fail("FAIL. Info toast alert coordinate positions do not match.");
+			}
+
+			takeScreenshot();
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase:312]: Use `OcularAssertElementImage` here and remove `takeScreenshot`

**Review by mateusralv — CHANGES_REQUESTED** [review]
Good Job! 😄

**Review by mateusralv — APPROVED** [review]

**Pull request closed without merging**